src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.ps1

```diff
@@ -187,6 +187,95 @@ function Test-subnetCRUD
     }
 }
 
+<#
+.SYNOPSIS
+Tests creating new simple virtualNetwork and subnets.
+.DESCRIPTION
+SmokeTest
+#>
+function Test-multiPrefixSubnetCRUD
```

Comment:
you'll need to add the test recording to the desktop test csproj file, [example](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.Netcore.csproj).